pkg/state/service.go

```diff
@@ -33,6 +33,30 @@ var serviceTableSchema = &memdb.TableSchema{
 // ServicesCollection stores and indexes Kong Services.
 type ServicesCollection collection
 
+func (k *ServicesCollection) AddIgnoringDuplicates(service Service) error {
```

Comment:
Would be nice to add a godoc for this `func` (actually there should be a linter catching that 🤔 ) explaining what it does and what's the use case.

Ditto for all the other `AddIgnoringDuplicates` funcs.